tests/io/test_pyarrow.py

```diff
@@ -256,6 +257,14 @@ def test_raise_on_opening_a_local_file_not_found() -> None:
         assert "[Errno 2] Failed to open local file" in str(exc_info.value)
 
 
+def test_pickle_pyarrow_file_io() -> None:
```

Comment:
Let me add a test for fsspec as well

Reply:
Yes, that would be great. You can just re-use an integration-test 👍

Reply:
Sorry for the delay on this, got busy with other work, Updated!